src/main/java/be/seeseemelk/mockbukkit/WorldMock.java

```
@@ -697,6 +773,12 @@ public Collection<Entity> getNearbyEntities(Location location, double x, double
 		throw new UnimplementedOperationException();
 	}
 
+	@Override
+	public @Nullable Entity getEntity(@NotNull UUID uuid)
+	{
+		return null;
```

Should probably throw an exception